pkg/infra/provider.go

```diff
@@ -388,3 +395,42 @@ func (p *AWSProvider) getClusterSecret(ctx context.Context) (*corev1.Secret, err
 
 	return secret, nil
 }
+
+func (p *AWSProvider) reconcileKubeconfig(ctx context.Context) error {
+	scopeCluster := p.scope
+
+	cluster := &clusterv1alpha1.Cluster{}
```

Comment:
user cluster as part of constructure, everything of update will be handle in https://github.com/kurator-dev/kurator/blob/8701068f74259ce5fa762b6d6e7cd0c278db542a/pkg/controllers/infra/cluster_controller.go#L89

Reply:
done